src/pages/debrief/components/vehicle-checks-card/__tests__/vehicle-checks-card.spec.ts

```diff
@@ -87,6 +100,84 @@ describe('VehicleChecksCardComponent', () => {
+          expect(showMeQuestionText.innerHTML.trim()).toBe('Show me question - Driving fault');
+        });
+        it('should indicate a tell me driving fault in Welsh for a Welsh test', (done) => {
+          fixture.detectChanges();
+          store$.dispatch(new ShowMeQuestionDrivingFault());
+          // Language change handled by parent page component, force the switch
+          translate.use('cy').subscribe(() => {
+            fixture.detectChanges();
+            const showMeQuestionText = fixture.debugElement.query(By.css('#show-me-question-outcome')).nativeElement;
+            expect(showMeQuestionText.innerHTML.trim()).toBe('[CY] Show me question - [CY] Driving fault');
+            done();
+          });
+        });
+        it('should indicate when there was a serious fault on the show me question', () => {
+          fixture.detectChanges();
+          store$.dispatch(new ShowMeQuestionSeriousFault());
+          fixture.detectChanges();
+          const showMeQuestionText = fixture.debugElement.query(By.css('#show-me-question-outcome')).nativeElement;
+          expect(showMeQuestionText.innerHTML.trim()).toBe('Show me question - Serious fault');
+        });
+        it('should indicate a tell me serious fault in Welsh for a Welsh test', (done) => {
+          fixture.detectChanges();
+          store$.dispatch(new ShowMeQuestionSeriousFault());
+          // Language change handled by parent page component, force the switch
+          translate.use('cy').subscribe(() => {
+            fixture.detectChanges();
+            const showMeQuestionText = fixture.debugElement.query(By.css('#show-me-question-outcome')).nativeElement;
+            expect(showMeQuestionText.innerHTML.trim()).toBe('[CY] Show me question - [CY] Serious fault');
+            done();
+          });
+        });
+        it('should indicate when there was a dangerous fault on the show me question', () => {
+          fixture.detectChanges();
+          store$.dispatch(new ShowMeQuestionDangerousFault());
+          fixture.detectChanges();
+          const showMeQuestionText = fixture.debugElement.query(By.css('#show-me-question-outcome')).nativeElement;
+          expect(showMeQuestionText.innerHTML.trim()).toBe('Show me question - Dangerous fault');
+        });
+        it('should indicate a tell me dangerous fault in Welsh for a Welsh test', (done) => {
+          fixture.detectChanges();
+          store$.dispatch(new ShowMeQuestionDangerousFault());
+          // Language change handled by parent page component, force the switch
+          translate.use('cy').subscribe(() => {
+            fixture.detectChanges();
+            const showMeQuestionText = fixture.debugElement.query(By.css('#show-me-question-outcome')).nativeElement;
+            expect(showMeQuestionText.innerHTML.trim()).toBe('[CY] Show me question - [CY] Dangerous fault');
+            done();
+          });
+        });
+      });
+    });
```

Comment:
could reference the translation json and test against that (e.g lines 157 and 175). Would make the tests less brittle - e.g would still work when we change the text to real welsh, or should the wording change in future.

Reply:
Good suggestion, done.